### PR TITLE
Fix date-input default example

### PR DESCRIPTION
**Pull request by igloosi**
BEFORE
screen shot 2018-03-28 at 10 33 56

AFTER
screen shot 2018-03-28 at 10 34 02

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ Breaking changes:
   within that block inline.
   (PR [#607](https://github.com/alphagov/govuk-frontend/pull/607))
 
+Fixes:
```
Comment (nickcolley) [inline CHANGELOG.md]: I think this is technically an internal change since we don't ship the yaml files.

File: src/date-input/template.test.js [hunk 2]
```diff
@@ -205,7 +205,6 @@ describe('Date input', () => {
   it('passes through fieldset params without breaking', () => {
     const $ = render('date-input', examples['default'])
 
-    expect(htmlWithClassName($, '.govuk-c-error-message')).toMatchSnapshot()
```
Comment (36degrees) [inline src/date-input/template.test.js]: Should this have moved or been split out into a different test rather than just being removed?

Comment (nickcolley) [reply src/date-input/template.test.js]: Hmm, yeah, I'll do a PR.

Comment (nickcolley) [reply src/date-input/template.test.js]: See https://github.com/alphagov/govuk-frontend/pull/625

Comment (igloosi) [reply src/date-input/template.test.js]: sorry

Comment (nickcolley) [reply src/date-input/template.test.js]: S'all good, slipped my mind too 👍

**Pull request merged**